scripts/kind-e2e/e2e.sh

```diff
@@ -51,6 +51,7 @@ with_context cluster3 subctl_install_subm
 
 with_context cluster2 deploy_resource "${RESOURCES_DIR}/netshoot.yaml"
 with_context cluster3 deploy_resource "${RESOURCES_DIR}/nginx-demo.yaml"
+[[ "${lighthouse}" = "true" ]] && with_context cluster3 deploy_resource "${DAPPER_SOURCE}/scripts/kind-e2e/nginx-demo-export.yaml"
```

Comment:
You may have to add a sleep before testing discovery in case it tries and gives up before cr can be distributed to clusters.

Reply:
I wonder how that wasn't failing before... :? , but you are right